src/components/Home.module.css

```diff
@@ -0,0 +1,106 @@
+.HomeTitle {
+  margin-top: 200px;
+  font-size: 2.5rem;
+}
+
+.HomeQuestion {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  min-height: 48px;
+  font-size: 3rem;
+  text-align: center;
+  margin: 30px 0 30px 200px;
+  background-color: transparent;
+  white-space: nowrap;
+}
+
+.NameInput {
+  font-size: 30px;
```

Comment:
The font size on the rest of this line is 48px.  Are you sure you want it smaller?